cmd/pipeline/deploy.go

```diff
@@ -244,7 +259,9 @@ func (pc *Command) waitUntilRunning(ctx context.Context, name string, action *ty
 
 		oktetoLog.Spinner("Waiting for containers to be healthy...")
 		exit <- pc.waitForResourcesToBeRunning(ctx, name, timeout)
-	}()
+	}(&wg)
+
+	wg.Wait()
```

Comment:
Why are we waiting for the 2 goroutines to finish? 
This will change the behavior, since before we weren't waiting for the old goroutine

Reply:
I think we were waiting before on the select but right now we are waiting even if the user press `CTRL+C`

Reply:
we were waiting before because there where the channels in place to listen to the events of exit and stop. indeed :) is the wait implementation for this flag of the pipeline :) i will take a look and give an other thought on this. thnks!